activity/journal.go

```diff
@@ -103,7 +103,7 @@ func (j Journal) Warnf(msg string, a ...interface{}) {
 // appends the message to that journal. Journals are stored in the user's cache directory under
 // `wash/activity/ID.log`.
 func (j Journal) Record(msg string, a ...interface{}) {
-	log.Debugf(msg, a...)
+	log.Printf(msg, a...)
```

Comment:
Won't this cause a lot of noise to the shell experience (e.g. seeing info messages when I'm doing a wash exec)?

Reply:
Wash the shell defaults to `warn` log level. The commit message talks about the rationale a bit more.

Reply:
The one area it could be messy is if we do #302, but in that case I think activity journal tracking becomes more complicated, and we need to send the logs back to the client that originated them anyway.

Reply:
Hmm, ok. When I do `./wash`, I still see INFO messages popping up when I try to do stuff (including server start up messages). This is from the latest Wash build.

Reply:
Do you have `loglevel: info` set in your config?

Reply:
Ah that's why. Cool, I don't have any other concerns around this.